packages/app/src/api/forest.ts

```diff
@@ -0,0 +1,202 @@
+import { PublicKey } from "@solana/web3.js";
+import { addUp, round } from "../common/utils";
+
+const MONGODB_API_URL = process.env.MONGODB_API_URL ?? "";
+const MONGODB_READ_TOKEN = process.env.MONGODB_READ_TOKEN ?? "";
```

Comment:
Does this work, I thought CRA env vars must be prefixed with REACT_APP_ ?

Reply:
yeah I forgot - fixed